src/main/java/org/broadinstitute/hellbender/tools/gvs/ingest/VetCreator.java

```diff
@@ -64,11 +66,6 @@ public VetCreator(String sampleIdentifierForOutputFileName, String sampleId, Str
     public void apply(VariantContext variant) throws IOException {
         int start = variant.getStart();
         long location = SchemaUtils.encodeLocation(variant.getContig(), start);
-        List<String> row = createRow(
-                location,
-                variant,
-                sampleId
-        );
```

Comment:
TOL perhaps rather than deleting this and inlining the `createRow` on line 80 this could just be moved into the `TSV` case?

Reply:
Right, I was suggesting to keep this block explicit but only in the `case TSV:` since that's the only spot it's used.

Reply:
👍